Automattic/wp-calypso
# AnchorFM: Add theme names and screenshots

#### Changes proposed in this Pull Request

- Adds two themes to the AnchorFM onboarding experience

#### Testing instructions

1. Apply this diff D54416-code
2. Checkout this branch
3. Go to http://calypso.localhost:3000/new?anchor_podcast={anchor_podcast_id}
4. Enter a podcast name
5. The available design screenshots should match the Figma designs
6. Clicking on one of the available designs should take you to a preview of those designs

#### Outstanding Items (Questions noted in the issue)

Generally unsure of the approach here so I'm welcoming feedback

- [ ] Screenshots for these themes need to be reproduced when designs are finalized
- [x] Blog sticker needs to be applied per D54416-code
- [x] Categories for these themes need to be determined

#### Screenshots
Available Design Screenshots
<img width="951" alt="Screen Shot 2020-12-16 at 7 58 41 PM" src="https://user-images.githubusercontent.com/66652282/102442663-ef0a5780-3ff2-11eb-8c92-ff0c737af8b3.png">

Design Preview
<img width="938" alt="Screen Shot 2020-12-16 at 7 58 47 PM" src="https://user-images.githubusercontent.com/66652282/102442670-f2054800-3ff2-11eb-85c9-f6a3b09abc02.png">

Fixes 372-gh-Automattic/dotcom-manage
